pkg/kubelet/volumemanager/cache/actual_state_of_world.go

```diff
@@ -323,6 +330,41 @@ func (asw *actualStateOfWorld) RemoveVolumeFromReportAsAttached(volumeName v1.Un
 	return nil
 }
 
+// Marks the specified volume as being set up
+func (asw *actualStateOfWorld) MarkVolumeAsBeingSetUp(volumeName v1.UniqueVolumeName) {
+	asw.Lock()
+	defer asw.Unlock()
+
+	volumeObj, volumeExists := asw.attachedVolumes[volumeName]
```

Comment:
is it ok to just ignore if volume doesn't exist?

Reply:
It's not possible to SetUp a volume that is not marked as attached in the ASW (it shouldn't be even mounted).

Reply:
so why check volumeExists? If it doesn't exist shouldn't we throw an error? I would prefer that than having this function fail silently

Reply:
OK then. I'll fix this.